scripts/regenerate-dbsf-oracle.py

```
@@ -249,14 +274,67 @@ def main() -> int:
+    with FIXTURE_PATH.open("r", encoding="utf-8") as fh:
+        actual = fh.read()
+    if actual == expected:
+        print(f"check PASS: {FIXTURE_PATH.relative_to(REPO_ROOT)} matches real-qdrant output.")
+        return 0
+
+    # Diff per-query to surface which query/score diverges.
+    try:
+        expected_obj = json.loads(expected)
+        actual_obj = json.loads(actual)
+        diffs: list[str] = []
+        for eq, aq in zip(expected_obj.get("queries", []), actual_obj.get("queries", [])):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)